doc/Uplink.xml

```diff
@@ -331,38 +331,75 @@
     <title>Configuration Interface</title>
     <section>
       <title>Configuration parameters</title>
-      <itemizedlist>
-        <listitem>
-          <para>RemoteAddress : Uniform resource locator by which the remote client can be
-            reached.</para>
-        </listitem>
-        <listitem>
-          <para>CertificateID : ID of the certificate to be used for client authentication.</para>
-        </listitem>
-        <listitem>
-          <para>UserLevel : Authorization level that will be assigned to the uplink
-            connection</para>
-        </listitem>
-        <listitem>
-          <para>Status :  Current connection status</para>
-        </listitem>
-         <listitem>
-          <para>CertPathValidationPolicyID : ID of Certificate Validation policy to validate the
-            remote uplink server certficate. If not configured, server certificate shall not be
-            validated. </para>
-        </listitem>
-        <listitem>
-          <para>AuthorizationServer :  JWTConfiguration token referring to an Authorization server
-            that provides JWT token to authorize with the uplink server.</para>
-        </listitem>
-        <listitem>
-          <para>Error : Optional user readable error information (readonly).</para>
-        </listitem>
-      </itemizedlist>
+      <table xml:id="table_uplink-config-parameters">
+        <title>Parameters description</title>
+        <tgroup cols="2">
+          <colspec colname="c1" colwidth="30*"/>
+          <colspec colname="c2" colwidth="70*"/>
+          <thead>
+            <row>
+              <entry>
+                <para>Parameter</para>
+              </entry>
+              <entry>Description</entry>
+            </row>
+          </thead>
+          <tbody valign="top">
+            <row>
+              <entry>RemoteAddress</entry>
+              <entry>
+                <para>Uniform resource locator by which the remote client can be reached.</para>
+              </entry>
+            </row>
+            <row>
+              <entry>
+                <para>CertificateID</para>
```

Comment:
Cosmetic, but all other fields do not have a <para>

Reply:
agreed, removed.